doc/source/visualization.rst

```diff
@@ -1779,38 +1848,12 @@ If the first grouping attribute is not specified the plots will be arranged in a
 
    plt.close('all')
 
-As shown above, scatter plots are also possible. Scatter plots allow you to map various data attributes to graphical properties of the plot. In the example above the colour and shape of the scatter plot graphical objects is mapped to 'day' and 'size' attributes respectively. You use scale objects to specify these mappings. The list of scale classes is given below with initialization arguments for quick reference.
-
-
-Scales
-~~~~~~
-
-::
-
-   ScaleGradient(column, colour1, colour2)
-
-This one allows you to map an attribute (specified by parameter column) value to the colour of a graphical object. The larger the value of the attribute the closer the colour will be to colour2, the smaller the value, the closer it will be to colour1.
-
-::
-
-   ScaleGradient2(column, colour1, colour2, colour3)
-
-The same as ScaleGradient but interpolates linearly between three colours instead of two.
-
-::
-
-   ScaleSize(column, min_size, max_size, transform)
-
-Map attribute value to size of the graphical object. Parameter min_size (default 5.0) is the minimum size of the graphical object, max_size (default 100.0) is the maximum size and transform is a one argument function that will be used to transform the attribute value (defaults to lambda x: x).
-
-::
-
-   ScaleShape(column)
-
-Map the shape of the object to attribute value. The attribute has to be categorical.
+This can also be done in ``seaborn``, at least for 3 variables:
 
-::
+.. code-block:: python
 
-   ScaleRandomColour(column)
+    g = sns.FacetGrid(tips_data, row="sex", col="smoker", hue="day")
```

Comment:
Are the different symbols then additional to the different colors? Or in place of?